### PR TITLE
Adding a new author: Mingyang Shi

### PR DESCRIPTION
**Pull request by beaioun**
### Description
Adding a new author for OSCI 2023: Mingyang Shi
I used a different email from the email I used to register for this program (mmyyshi@gmail.com). The email I used to sign the commit was my github account email, which is mshi@ucsd.edu
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/mingyang-shi.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Mingyang Shi
+short_name: Will
+github: beaioun
+---
+
+**Mingyang Shi** Hi, I am a Data Scientist/Engineer with AWS Solution Architect certified. My job is to deal with different sources of data and orchestrate a system and dashboard to show to the end users. I am activly seeaking new opportunities to improve myself and advance my career. I am glad you find my work meaningful! Thank you for visiting my page!
```
Comment (cwperks) [inline _authors/osci_2023/mingyang-shi.markdown:7]: nit: actively seeking*

**Review by cwperks — APPROVED** [review]
LGTM!

File: _authors/osci_2023/mingyang-shi.markdown (hunk 1 above)
Comment (beaioun) [reply _authors/osci_2023/mingyang-shi.markdown:7]: Sorry I'll recommit! A little bit rusty to type without grammarly.

**Review by dtaivpp — APPROVED** [review]
Approved!

**Pull request merged**